### PR TITLE
Feature/team

### PR DESCRIPTION
**Pull request by Mercy-Chemtai**
# Engineers Checklist

## What this PR does?
- List of updates for features, enhancements, or bugfixes

## How can it be tested?
- Please provide step-by-step guide on how to test
- If web or mobile, screenshots will be very helpful in addition

# Where is it documented?
- List JIRA tickets

## Backend Updates
- [ ] Updated Postman Documentation
	- [ ] Requests
	- [ ] Examples
	- [ ] Tests
- [ ] Updated Tasks on JIRA
- [ ] Tested API updates in `dev` environment
- [ ] CI Checks Passed

### CODE REVIEW

**Review by faith-hopewachira — APPROVED** [review]

**Review by Mutesi-alin — APPROVED** [review]

**Review by aine-mbabazi — APPROVED** [review]

**Review by raziahabdull — APPROVED** [review]

File: mitushconnect/src/App.js [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
+ feature/team
+import Team from './Team';
+
+ 
 
 const App = () =>{
   return(
     <div>
         <Services/>
+ feature/team
```
Comment (hunterachieng) [inline mitushconnect/src/App.js:13]: What is this?

File: mitushconnect/src/App.js [hunk 2]
```diff
@@ -1,14 +1,28 @@
-import logo from './logo.svg';
+ HEAD
 import './App.css';
 import Services from "./OurServices"
+ feature/team
```
Comment (hunterachieng) [inline mitushconnect/src/App.js:4]: What is this?

File: mitushconnect/src/OurServices/index.js [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+import './index.css'
+function OurServices(){
+    return(
+        <div className="div-container">
+            <h1 className='services-heading'>Our Services</h1>
+        <div className='cardsContainer'>
+        <div className="cardsText" id='delivery-card'>
+            <h1 className="h1Tag">Pick-up</h1>
 import './index.css'
 function OurServices(){
```
Comment (hunterachieng) [inline mitushconnect/src/OurServices/index.js:11]: What is happening here?

File: mitushconnect/src/Team/index.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            <img src="/Images/Chemtai.jpeg" alt="Chemtai" className='Chemtai'  />
+            <div className='figure'>
+            <p className="software"> Mercy Chemtai</p>
+            <p className="software">Software Developer</p>
+            </div>
+            </div>
+            <div>
+            <img src="/Images/Faith.jpeg" alt="Faith" className='Faith'/>
+            <div className='figure'>
+            <p className="software"> Faith Wachira</p>
```
Comment (hunterachieng) [inline mitushconnect/src/Team/index.js:33]: Why would you call these software?

**Review by Mercy-Chemtai — COMMENTED** [review]
Done

File: mitushconnect/src/App.js [hunk 5]
```diff
@@ -1,14 +1,25 @@
-import logo from './logo.svg';
+ HEAD
```
Comment (hunterachieng) [inline mitushconnect/src/App.js:1]: What is this?

File: mitushconnect/src/OurServices/index.js [hunk 6]
```diff
@@ -1,5 +1,12 @@
+
 import './index.css'
 function OurServices(){
+    return(
```
Comment (hunterachieng) [inline mitushconnect/src/OurServices/index.js:4]: What is happening here

File: mitushconnect/src/App.js [hunk 7]
```diff
@@ -1,14 +1,21 @@
...
 
 const App = () =>{
   return(
     <div>
         <Services/>
+        <Team/>
     </div>
   )
+
+}
```
Comment (hunterachieng) [inline mitushconnect/src/App.js:16]: You have extra brackets here

File: mitushconnect/src/OurServices/index.js [hunk 8]
```diff
@@ -13,6 +19,19 @@ function OurServices(){
             Save time with our errand services where we pick up your bales at your preferred location.
             </p>
         </div>
+        <div className="cardBox" id='delivery-card'>
+            <h1 className="h1Tag">Company</h1>
```
Comment (hunterachieng) [inline mitushconnect/src/OurServices/index.js:23]: Why would a heading be called `h1Tag`?

**Review by hunterachieng — APPROVED** [review]

**Review by raziahabdull — APPROVED** [review]

**Review by Mutesi-alin — APPROVED** [review]
h

**Review by faith-hopewachira — APPROVED** [review]

**Review by aine-mbabazi — APPROVED** [review]

**Pull request merged**